src/expansion_context.mli

```diff
@@ -53,6 +53,9 @@ module Extension : sig
   val code_path : t -> Code_path.t
   (** Return the code path for the given context *)
 
+  val input_name : t -> string
```

Comment:
What do you think about using the same documentation as in `Base.input_name`? And also for `code_path` using the same as in `Base.code_path`? The long documentation for those two functions in `Base` is to make clear what the difference between `input_name` and `code_path.file_path` is.

Reply:
Done, I literally copied the documentation (I hope that's what you had in mind)